ui/lib/config-ui/addon/routes/messages/create.js

```diff
@@ -22,4 +22,14 @@ export default class MessagesCreateRoute extends Route {
       authenticated,
     });
   }
+
+  setupController(controller, resolvedModel) {
+    super.setupController(controller, resolvedModel);
+
+    const authenticated = resolvedModel.authenticated === undefined ? true : resolvedModel.authenticated;
```

Comment:
I think, but double check, this can be `const authenticated = !!resolvedModel.authenticated` instead